Homework7/lesson6/my_weather.py

```diff
@@ -0,0 +1,39 @@
+from requests import get
+import json
+import sys
+
+nargs = len(sys.argv)
+#print(nargs)
+
+if nargs == 1:
+    try:
+        city_name = sys.argv[1]
+    except:
+        print("error: the city does not exists... please provide city (Paris? Dublin?) " \
+              "on the command line")
+        sys.exit(1)  # abort
+elif nargs == 2:
+    city_names = sys.argv[1]
+    cities = city_names.split(',')
+    for city_name in cities:
+        city = get("http://api.weatherstack.com/current?access_key=0823fd35d0df5ccec0e8eb6bd3660635&query={}%20".format(
```

Comment:
You should remove the token from the code as this is your personal token

Reply:
Can you advise on how? i tired several options (including dotenv - none worked...) can you share insights? thanks!

Reply:
Done. Next time you check please export an env var. I've inserted a note for it.